variables.tf

```diff
@@ -124,7 +124,7 @@ variable "health_check" {
 
 variable "volume" {
   description = "A set of volume blocks that containers in your task may use."
-  type        = list(string)
+  type        = list(map(string))
```

Comment:
I think that you could simply change:

```hcl
type        = list(string)
```

to

```hcl
type        = list(any)
```

That way you would not need to remove the code task.tf:41-51.

Reply:
The resource that was changed `"aws_ecs_task_definition" "fargate"` requires the FARGATE launch type.  It was my understanding that DockerVolumeConfiguration only works with EC2 launch type, along with EFS.  Is this not true anymore?

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-volumes.html#specify-volume-config

```
dockerVolumeConfiguration
Type: Object

Required: No

This parameter is specified when using Docker volumes. Docker volumes are only supported when using the EC2 launch type. Windows containers only support the use of the local driver. To use bind mounts, specify a host instead.
```

Reply:
@JonRoma and @dimpsey EFS support for Fargate is on the roadmap:

https://github.com/aws/containers-roadmap/projects/1

Based on that and the tag it appears that it will be released in preview mode soonish:

https://github.com/aws/containers-roadmap/issues/53

I recall reading an announcement in January that a preview release was going to be released in the next six months. I can't find the announcement anymore.